dwave/system/composites/parallel_embeddings.py

```diff
@@ -283,31 +294,94 @@ def sample(self, bqm, **kwargs):
         Examples:
             See class examples.
         """
+        if chain_strength is None:
+            chain_strength = None
+        chain_strengths = [chain_strength] * self.num_embeddings
+        bqms = [bqm] * self.num_embeddings
+        if "initial_state" in kwargs:
+            kwargs["initial_states"] = [
+                kwargs.pop("initial_state")
+            ] * self.num_embeddings
+
+        responses, info = self.sample_as_list(bqms, chain_strengths, **kwargs)
+
+        if self.num_embeddings == 1:
+            return responses[0]
+        else:
+            answer = dimod.concatenate(responses)
+            answer.info.update(info)
+            return answer
+
+    def sample_as_list(
+        self,
+        bqms: list[dimod.BinaryQuadraticModel],
+        chain_strengths: Optional[list] = None,
+        **kwargs
+    ) -> tuple[list[dimod.SampleSet], dict]:
+        """Sample from the specified binary quadratic models.
+
+        Samplesets are returned for every embedding, the binary quadratic model
+        solved on each embedding needn't be identical.
+        kwargs are passed unmodifiedto the child sampler, with the exception of
+        `initial_states` (one state per embedding) which is composed to a
+        single `initial_state` parameter for the child sampler analogous to
+        the bqm composition.
+
+        Args:
+            bqms:
+                Binary quadratic models to be sampled from. A list that
+                should be ordered to match self.embeddings.
+
+            chain_strengths:
+                The chain strength parameters for each bqm. A list that
+                should be ordered to match self.embeddings.
+
+            **kwargs:
+                Optional keyword arguments for the sampling method, specified per solver.
+
+        Returns:
+            A typle consisting of:
+            1. A list of :class:`~dimod.SampleSet`, one per embedding
+            2. The info field returned by the child sampler
+        Examples:
+            See class examples.
+        """
 
         # apply the embeddings to the given problem to tile it across the child sampler
-        embedded_bqm = dimod.BinaryQuadraticModel.empty(bqm.vartype)
+        embedded_bqm = dimod.BinaryQuadraticModel.empty(bqms[0].vartype)
 
         __, __, target_adjacency = self.target_structure
-        for embedding in self.embeddings:
+        if chain_strengths is None:
+            chain_strengths = [None] * self.num_embeddings
```

Comment:
Similarly here, would it make sense having empty `chain_strengths` be `None`?

```suggestion
        if not chain_strengths:
            chain_strengths = [None] * self.num_embeddings
```